probes/releasesAreSigned/impl.go

```diff
@@ -53,12 +53,12 @@ func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
 
 	totalReleases := 0
 	for releaseIndex, release := range releases {
```

Comment:
Maybe neater to use a slice here?
```go
for releaseIndex, release := range releases[:releaseLookBack] {
```

Reply:
that could unfortunately panic
https://go.dev/play/p/8zDplRlC5Yb